public/app/features/alerting/unified/rule-list/RuleListPageTitle.tsx

```diff
@@ -40,8 +41,8 @@ export function RuleListPageTitle({ title }: { title: string }) {
       <h1>{title}</h1>
       {shouldShowV2Toggle && (
         <div>
-          <Button size="sm" fill="outline" {...buttonConfig} onClick={toggleListView} className="fs-unmask">
-            {buttonConfig.text}
+          <Button size="sm" fill="outline" {...configToUse} onClick={toggleListView} className="fs-unmask">
```

Comment:
Avoiding spreading `text` onto the element - wasn't doing any harm, but was likely unintentional?

Reply:
yes!